test/index.js

```diff
@@ -90,6 +89,37 @@ describe('lune', function () {
       )
     })
   })
+
+  describe('#phase_range', function () {
+    const NAMES = ['new', 'q1', 'full', 'q3']
+
+    /* http://aa.usno.navy.mil/data/docs/JulianDate.php */
+    it('should return all moon phases within a time range', function () {
+      let error = 0
+
+      for (let name of NAMES) {
+        const actual = lune.phase_range(
+          new Date('1989-01-01T00:00Z'),
+          new Date('1989-10-01T00:00Z'),
+          name
+        )
+        const expected = observations.filter(function (x) {
+          return x[2] === name
+        })
+        assert.strictEqual(actual.length, expected.length)
+
+        for (let i = 0; i < actual.length; i++) {
```

Comment:
Can't: in this case, we're reading from two arrays simultaneously, so we need an index so we can reference them both.